src/search-app.js

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+
+async function fetchCountryInformation(country) {
+    try {
+        const countryUrl = "https://restcountries.com/v2/name/" + country;
+        const countryInfo = await axios.get(countryUrl);
+        console.log(countryInfo.data);
+        createCountryInfo(countryInfo.data);
+    } catch (e) {
+        console.error(e);
+        countryNotFound();
+    }
+}
+
+function countryNotFound() {
+    const warningText = document.getElementById("country-not-found-warning");
+    const countryInfo = document.getElementById("country-info-found");
+    warningText.innerText = "Country not found";
+    countryInfo.innerText = "";
+}
+
+function countryCurrency(country) {
+    let currencyText = "";
+    for (let i = 0; i < country.currencies.length; i++) {
+        if (i > 0 && i < country.currencies.length) {
+            currencyText += " and ";
+        }
+        currencyText += country.currencies[i].name;
+    }
+    return currencyText;
```

Comment:
Als je dit resultaat wilt bereiken dan kun je dit simpeler doen:
`return country.currencies.join(" and ")`